std_msgs/Header header
bool reduce_velocity
float32 target_dist
float32 steering_dir
